ts/src/okx.ts

```diff
@@ -5112,6 +5113,136 @@ export default class okx extends Exchange {
         return this.filterByArrayPositions (result, 'symbol', symbols, false);
     }
 
+    async fetchPositionsForSymbol (symbol, params = {}) {
+        await this.loadMarkets ();
```

Comment:
@ttodua can you add JSdocs for this method and the base method in Exchange.ts with the proper return type? `Promise<Position[]>`

Reply:
sure, 1 min